docs/tutorial/conversion.rst

```diff
@@ -296,4 +296,207 @@ Create the database and start the application::
+        with hotness.cache.cache_lock:
+            if not hotness.cache.cache.is_configured:
+                hotness.cache.cache.configure(**self.config["cache"])
+
+        self.mdapi_url = self.config.get("mdapi_url")
+        _log.info("Using hotness.mdapi_url=%r" % self.mdapi_url)
+        self.repoid = self.config.get("repoid", "rawhide")
+        _log.info("Using hotness.repoid=%r" % self.repoid)
+        self.distro = self.config.get("distro", "Fedora")
+        _log.info("Using hotness.distro=%r" % self.distro)
+
+        # Retrieve the requests configuration; by default requests time out
+        # after 15 seconds and are retried up to 3 times.
+        self.requests_session = requests.Session()
+        self.timeout = (
+            self.config.get("connect_timeout", 15),
+            self.config.get("read_timeout", 15),
+        )
+        retries = self.config.get("hotness.requests_retries", 3)
+        retry_conf = retry.Retry(
+            total=retries, connect=retries, read=retries, backoff_factor=1
+        )
+        retry_conf.BACKOFF_MAX = 5
+        self.requests_session.mount(
+            "http://", requests.adapters.HTTPAdapter(max_retries=retry_conf)
+        )
+        self.requests_session.mount(
+            "https://", requests.adapters.HTTPAdapter(max_retries=retry_conf)
+        )
+        _log.info(
+            "Requests timeouts are {}s (connect) and {}s (read)"
+            " with {} retries".format(self.timeout[0], self.timeout[1], retries)
+        )
+
+        # Build a little store where we'll keep track of what koji scratch
+        # builds we have kicked off.  We'll look later for messages indicating
+        # that they have completed.
+        self.scratch_builds = {}
+
+        _log.info("That new hotness ticket filer is all initialized")
+
+Wrapper function
+----------------
+The next step is to change ``consume`` method to ``__call__`` method. This is pretty
+straightforward. After this modification ``__call__`` method should look like this::
+
+    def __call__(self, msg):
+        """
+        Called when a message is received from queue.
+
+        Params:
+            msg (fedora_messaging.message.Message) The message we received
+                from the queue.
+        """
+        topic, body, msg_id = msg.topic, msg.body, msg.id
+        _log.debug("Received %r" % msg_id)
+
+        if topic.endswith("anitya.project.version.update"):
+            self.handle_anitya_version_update(msg)
+        elif topic.endswith("anitya.project.map.new"):
+            self.handle_anitya_map_new(msg)
+        elif topic.endswith("buildsys.task.state.change"):
+            self.handle_buildsys_scratch(msg)
+        else:
+            _log.debug("Dropping %r %r" % (topic, body))
+            pass
+
+In this case we are working with the message using the standard
+``fedora_messaging.message.Message`` methods. It is always better to use schema specific
+methods for any topic you are receiving.
+
+Testing
```

Comment:
I think it would be better if most of this was replaced with a link to the hotness docs and a sentence like "In a [configured development environment](link to docs), we can easily test our conversion by..."